sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -909,12 +910,14 @@ public void setName(@NotNull String name, @NotNull TransactionNameSource transac
 
   @Override
   public @Nullable ISpan getLatestActiveSpan() {
-    final List<Span> spans = new ArrayList<>(this.children);
-    if (!spans.isEmpty()) {
-      for (int i = spans.size() - 1; i >= 0; i--) {
-        if (!spans.get(i).isFinished()) {
-          return spans.get(i);
-        }
+    @NotNull final ListIterator<Span> iterator = this.children.listIterator();
+    while (iterator.hasNext()) {
```

Comment:
See above, we could use the new util `CollectionUtils.reverseListIterator(CopyOnWriteArrayList list)`